jerry-port/default/default-date.c

```diff
@@ -47,6 +66,25 @@ double jerry_port_get_local_time_zone_adjustment (double unix_ms,  /**< ms since
 #else /* !HAVE_TM_GMTOFF */
   (void) unix_ms;
   (void) is_utc;
+#ifdef _WINDOWS
+  FILETIME fileTime, localFileTime;
+  SYSTEMTIME systemTime, localSystemTime;
+  ULARGE_INTEGER time, localTime;
+
+  _tzset ();
+  UnixTimeToFileTime ((LONGLONG) (unix_ms / 1000), &fileTime);
+
+  if (FileTimeToSystemTime (&fileTime, &systemTime) &&
+      SystemTimeToTzSpecificLocalTime (0, &systemTime, &localSystemTime) &&
+      SystemTimeToFileTime (&localSystemTime, &localFileTime))
```

Comment:
Please move the `&&` operators in the following way.
```c
if (FileTimeToSystemTime (&fileTime, &systemTime) 
    && SystemTimeToTzSpecificLocalTime (0, &systemTime, &localSystemTime)
    && SystemTimeToFileTime (&localSystemTime, &localFileTime))
```

Reply:
Fixed.